lib/ansible/module_utils/docker_common.py

```diff
@@ -83,16 +83,16 @@
 DEFAULT_TIMEOUT_SECONDS = 60
 
 DOCKER_COMMON_ARGS = dict(
-    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST),
-    tls_hostname=dict(type='str', default=DEFAULT_TLS_HOSTNAME),
+    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST, fallback=(env_fallback, 'DOCKER_HOST')),
+    tls_hostname=dict(type='str', default=DEFAULT_TLS_HOSTNAME, fallback=(env_fallback, 'DOCKER_TLS_HOSTNAME')),
     api_version=dict(type='str', aliases=['docker_api_version'], default='auto'),
     timeout=dict(type='int', default=DEFAULT_TIMEOUT_SECONDS),
     cacert_path=dict(type='str', aliases=['tls_ca_cert']),
     cert_path=dict(type='str', aliases=['tls_client_cert']),
     key_path=dict(type='str', aliases=['tls_client_key']),
-    ssl_version=dict(type='str'),
-    tls=dict(type='bool', default=DEFAULT_TLS),
-    tls_verify=dict(type='bool', default=DEFAULT_TLS_VERIFY),
+    ssl_version=dict(type='str', default=DEFAULT_SSL_VERSION, fallback=(env_fallback, 'DOCKER_SSL_VERSION')),
+    tls=dict(type='bool', default=DEFAULT_TLS, fallback=(env_fallback, 'DOCKER_TLS_VERIFY')),
```

Comment:
The fallback env variable is called `DOCKER_TLS`, not `DOCKER_TLS_VERIFY` (that's the one below).